api/output_policy.go

```diff
@@ -44,8 +48,22 @@ func (d *OutputPolicyError) HCLString() (string, error) {
 
 // Builds a sample policy document from the request
 func (d *OutputPolicyError) buildSamplePolicy() (string, error) {
+	MethodStr := d.method
+	// List is often defined as a URL param instead of as an http.Method
+	// this will check for the header and properly switch off of the intended functionality
+	if d.params.Has(listKey) {
+		isList, err := strconv.ParseBool(d.params.Get(listKey))
+		if err != nil {
+			return "", err
```

Comment:
This error could probably be improved with some context. Alternatively you can simply check for `strings.Lower(d.params.Get("list")) == "true"` or something similar